pkg/ibmcsidriver/controller_helper.go

```diff
@@ -491,19 +522,19 @@ func overrideParams(logger *zap.Logger, req *csi.CreateVolumeRequest, config *co
 				volume.Region = value
 			}
 		case IOPS:
-			// Override IOPS only for custom or dp2
-			if volume.Capacity != nil && volume.VPCVolume.Profile != nil && volume.VPCVolume.Profile.Name == DP2Profile {
-				var iops int
-				var check bool
-				iops, err = strconv.Atoi(value)
-				if err != nil {
-					err = fmt.Errorf("%v:<%v> invalid value", key, value)
+			if len(value) != 0 {
+				logger.Info("override (IOPS)", zap.Any(IOPS, value))
+				iopsStr := value
+				volume.Iops = &iopsStr
+			}
+		case Throughput:
```

Comment:
Throughput is only supported for rfs profile, looks code is not uniform somewhere we are checking profile and taking action and somewhere its not

Reply:
This is how the flow is we parse all the values from storage class and then we do all validations. Do we want to change the current flow for this ?